packages/base/links-to-many-component.gts

```diff
@@ -241,28 +242,23 @@ export function getLinksToManyComponent({
     field: Field<typeof BaseDef>,
     boxedElement: Box<BaseDef>,
   ): typeof BaseDef;
-  context?: CardContext;
 }): BoxComponent {
   if (format === 'edit' || format === 'atom') {
     return class LinksToManyEditorTemplate extends GlimmerComponent {
       <template>
-        <LinksToManyEditor
-          @model={{model}}
-          @arrayField={{arrayField}}
-          @field={{field}}
-          @format={{format}}
-          @cardTypeFor={{cardTypeFor}}
-          @context={{context}}
-        />
+        <CardContextConsumer as |context|>
+          <LinksToManyEditor
+            @model={{model}}
+            @arrayField={{arrayField}}
+            @field={{field}}
+            @format={{format}}
+            @cardTypeFor={{cardTypeFor}}
+            @context={{context}}
```

Comment:
we should consume context via https://github.com/consume in LinksToManyEditor instead of passing it here